MimeKit/Cryptography/RsaEncryptionPadding.cs

```diff
@@ -243,25 +239,5 @@ internal AlgorithmIdentifier GetAlgorithmIdentifier ()
 
 			return new AlgorithmIdentifier (PkcsObjectIdentifiers.IdRsaesOaep, GetRsaesOaepParameters ());
 		}
-
-#if NETCOREAPP3_0
-		internal RSAEncryptionPadding AsRSAEncryptionPadding ()
-		{
-			switch (Scheme) {
-			case RsaEncryptionPaddingScheme.Oaep:
-				switch (OaepHashAlgorithm) {
-				case DigestAlgorithm.Sha1: return RSAEncryptionPadding.OaepSHA1;
-				case DigestAlgorithm.Sha256: return RSAEncryptionPadding.OaepSHA256;
-				case DigestAlgorithm.Sha384: return RSAEncryptionPadding.OaepSHA384;
-				case DigestAlgorithm.Sha512: return RSAEncryptionPadding.OaepSHA512;
-				default: return null;
-				}
-			case RsaEncryptionPaddingScheme.Pkcs1:
-				return RSAEncryptionPadding.Pkcs1;
-			default:
-				return null;
-			}
-		}
-#endif
```

Comment:
I might want to keep this block, but it should probably be `#if false` for now instead. Let me get back to you on this one... need to check the WindowsSecureMimeContext logic to see if this would be useful when converting MimeKit's CmsRecipient into a System.Security CmsRecipient. I *may* have a bug there if I don't have logic to do this kind of thing.

Reply:
👍

Reply:
I reverted the NETCOREAPP3_0 stuff locally because it looks like I should really be using this stuff (but obviously not #if'd as NETCOREAPP3_0 - it'll probably be #if NET5_0 or something).

Reply:
glad we uncovered this (dead code that shouldn't have been killed)